src/privatesend-client.cpp

```diff
@@ -1261,24 +1272,23 @@ bool CPrivateSendClient::CreateDenominated(const CompactTallyItem& tallyItem, bo
 
     LogPrintf("CreateDenominated0 nValueLeft: %f\n", (float)nValueLeft/COIN);
     // make our collateral address
-    CReserveKey reservekeyCollateral(pwalletMain);
-
-    CScript scriptCollateral;
-    CPubKey vchPubKey;
-    assert(reservekeyCollateral.GetReservedKey(vchPubKey, false)); // should never fail, as we just unlocked
-    scriptCollateral = GetScriptForDestination(vchPubKey.GetID());
-
-    // ****** Add collateral outputs ************ /
+    CReserveKey reservekeyCollateral(pwalletMain);    
```

Comment:
reservekeyCollateral can be also stored in:
CKeyHolderStorage keyHolderStorageDenom